### ISSUE TITLE
no docs for odoc :-(

### ISSUE THREAD

**Issue by jonludlam** [ocurrent/ocaml-docs-ci#170]
There are no docs for odoc on ocaml.org - this appears to be a problem with the solver - https://docs.ci.ocaml.org/package/odoc.2.4.1 shows

```
Error from solver: Bad frame from worker: time="1.36" len="0.0"
```

**Comment by tmcgilchrist** [comment]
That is an error from the solver in docs, rather than failing to find a solution for odoc. Reported under https://github.com/ocurrent/solver-service/pull/61. 
The fix would either need to be ported across to the copy of solver-service in this project or modify docs-ci to use the solver-service over CapnProto.

**Comment by jonludlam** [comment]
I'm not sure it's the same cause, because every single version of odoc is failing with the same error, and ocurrent/solver-service#61 looks less reproducible.

**Comment by moyodiallo** [comment]
> I'm not sure it's the same cause, because every single version of odoc is failing with the same error, and [ocurrent/solver-service#61](https://github.com/ocurrent/solver-service/pull/61) looks less reproducible.

Indeed, this is less reproducible, it was known when scaling solver-service which put pressure on the solver. Lot of issues pop up and we solved them. The solver was stable and could take more than the actual number of worker in "ocaml-docs-ci" which is 6, it was successfully tested with 128 workers.

The solver of ocaml-docs-ci could be upgrade to this commit https://github.com/ocurrent/solver-service/tree/4e161fc300bc39675630d4e1059c363dbcb9c50f just before the port on eio. Do not hesitate to DM me, I'm willing, even to review the code, I've good knowledge of that part of the code.

**Comment by shonfeder** [comment]
Seems like https://github.com/ocurrent/ocaml-docs-ci/issues/86 would obviate the need for this.

We need to make a decision on the role of the solver services, and then decide our priorities here accordingly.

**Comment by mtelvers** [comment]
I note that the solver service, which @talex5 has recently improved, is now very fast.  It would be great to get docs.ci.ocaml.org to use it.

**Comment by jonludlam** [comment]
Fixed now.
